### PR TITLE
[Process] Implement Groups() to get additional gids.

### PR DESCRIPTION
**Pull request by TakayukiBGoto**
I added Groups and fill it with additional gids. 
I think additional gids is useful information for some users.

### CODE REVIEW

**Comment by Lomanic** [comment]
Simply naming this function Groups has a non-negligible chance of confusing the users with the Gids function also. Maybe rename this new Groups function to SupplementaryGroups. Your POV @shirou?

**Comment by shirou** [comment]
I think `Groups` is ok. Because `id` command outputs `groups` which includes supplementary groups like this
```
%id
uid=1000(shirou) gid=1000(shirou) groups=1000(shirou),4(adm),24(cdrom),27(sudo),30(dip),132(libvirtd),998(docker)
```

and other reason, Groups in `/proc/status` includes not only supplementary groups, but also primary group.

**Comment by TakayukiBGoto** [comment]
Sorry for my late reply.
I fixed requested changes.

Could you decide whether to change function name or not?

**Comment by Lomanic** [comment]
Let's keep function name as it is.

**Comment by TakayukiBGoto** [comment]
@Lomanic Thank you for your reply.
I think I fixed all your requested change.
Please review again.

File: process/process_openbsd.go [hunk 1]
```diff
@@ -204,6 +204,18 @@ func (p *Process) GidsWithContext(ctx context.Context) ([]int32, error) {
 
 	return gids, nil
 }
+func (p *Process) Groups() ([]int32, error) {
```
Comment (Lomanic) [inline process/process_openbsd.go]: this is now defined twice for openbsd

Comment (TakayukiBGoto) [reply process/process_openbsd.go]: Sorry, I overlooked that.
I fixed.

**Comment by shirou** [comment]
Sorry for late and thank you for your contribution. Change request is already fixed, so I merge this.

**Pull request merged**